src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const peopleWithParents = people.map(person => {
+    const mother = people.find(mom => mom.name === person.motherName);
+    const father = people.find(dad => dad.name === person.fatherName);
+
+    return { ...person, mother, father };
+  });
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {peopleWithParents.map(person => (
+          <PersonLink key={person.slug} person={person} people={people} />
```

Comment:
Ensure that `person.slug` is a unique identifier for each person. Using non-unique keys can lead to performance issues and bugs in React. If `slug` is not guaranteed to be unique, consider using a different unique identifier.